service/repository/csv.go

```diff
@@ -0,0 +1,108 @@
+package repository
+
+import (
+	"bufio"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/etyberick/golang-bootcamp-2020/entity"
+)
+
+type quoteRepository struct {
+	file   os.File
+	quotes []entity.Quote
+}
+
+// EmptyDatabase message response
+const EmptyDatabase = "database is empty"
+
+// QuoteRepository manages all persistency management operations
+type QuoteRepository interface {
+	ReadAll() ([]entity.Quote, error) // Fetches all items from data source
+	Write(entity.Quote) error         // Writes an item into data source
```

Comment:
These comments are already present when defining the functions, right? Do you think we need them here?

Reply:
you're right... and done!